ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/ProductDataProvider.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api;
+
+import javax.validation.constraints.NotNull;
+
+import org.vectomatic.dom.svg.ui.SVGResource;
+
+/**
+ * Implementation will provide product information
+ *
+ * @author Alexander Andrienko
+ */
+public interface ProductDataProvider {
```

Comment:
Not sure that this is good name for this interface. Maybe it should be related to "packaging" or smth like that